test/qa/test_connect.py

```diff
@@ -24,11 +26,11 @@ def teardown_function(function):  # noqa: ARG001
     daemon.stop()
 
 
-def connect_base_test(connection_settings, group=(), name="", hostname=""):
+def connect_base_test(connection_settings, group=(), name="", hostname="", virtual=False):
```

Comment:
by default `Virtual-location` is true. Is is in sync with this param value?

Reply:
This is to denote that server/group we are connecting to is a virtual, not the setting value.